test/bootstrap_checkbox_test.rb

```diff
@@ -9,52 +9,52 @@ def setup
-    assert_equivalent_xml expected, @builder.check_box(:terms, {label: 'I agree to the terms'}, 'yes', 'no')
+    assert_equivalent_xml expected, @builder_new.check_box(:terms, {label: 'I agree to the terms'}, 'yes', 'no')
   end
 
   test "inline checkboxes" do
     expected = %{<label class="form-check-inline" for="user_terms"><input name="user[terms]" type="hidden" value="0" /><input class="form-check-input" id="user_terms" name="user[terms]" type="checkbox" value="1" /> I agree to the terms</label>}
-    assert_equivalent_xml expected, @builder.check_box(:terms, label: 'I agree to the terms', inline: true)
+    assert_equivalent_xml expected, @builder_new.check_box(:terms, label: 'I agree to the terms', inline: true)
   end
 
   test "disabled inline check_box" do
     expected = %{<label class="form-check-inline disabled" for="user_terms"><input disabled="disabled" name="user[terms]" type="hidden" value="0" /><input class="form-check-input" disabled="disabled" id="user_terms" name="user[terms]" type="checkbox" value="1" /> I agree to the terms</label>}
-    assert_equivalent_xml expected, @builder.check_box(:terms, label: 'I agree to the terms', inline: true, disabled: true)
+    assert_equivalent_xml expected, @builder_new.check_box(:terms, label: 'I agree to the terms', inline: true, disabled: true)
   end
 
   test "inline checkboxes with custom label class" do
     expected = %{<label class="form-check-inline btn" for="user_terms"><input name="user[terms]" type="hidden" value="0" /><input class="form-check-input" id="user_terms" name="user[terms]" type="checkbox" value="1" /> Terms</label>}
-    assert_equivalent_xml expected, @builder.check_box(:terms, inline: true, label_class: 'btn')
+    assert_equivalent_xml expected, @builder_new.check_box(:terms, inline: true, label_class: 'btn')
   end
 
   test 'collection_check_boxes renders the form_group correctly' do
     collection = [Address.new(id: 1, street: 'Foobar')]
-    expected = %{<input id="user_misc" multiple="multiple" name="user[misc][]" type="hidden" value="" /><div class="form-group"><label class="form-control-label" for="user_misc">This is a checkbox collection</label><div class="form-check"><label class="form-check-label" for="user_misc_1"><input class="form-check-input" id="user_misc_1" name="user[misc][]" type="checkbox" value="1" /> Foobar</label></div><span class="form-text text-muted">With a help!</span></div>}
+    expected = %{<input id="user_misc" multiple="multiple" name="user[misc][]" type="hidden" value="" /><div class="form-group"><label class="form-control-label" for="user_misc">This is a checkbox collection</label><div class="form-check"><label class="form-check-label" for="user_misc_1"><input class="form-check-input" id="user_misc_1" name="user[misc][]" type="checkbox" value="1" /> Foobar</label></div><small class="form-text text-muted">With a help!</span></div>}
```

Comment:
Here an a number of other places you changed the `<span>` to a `<small>`, but the closing tag is still a `</span>`.  I doubt that's causing your Travis problem, but it's worth noting.

Reply:
I fixed these issues with commit 6085f8c.
However, I needed to change more in test/test_helper.rb to get "rake test" for all versions working. Probably the branch bootstrap-v4 should include latest changes from "master" to resolve all Travis test issues?!